packages/jellyfish-api-core/src/category/poolpair.ts

```diff
@@ -201,6 +201,8 @@ export interface PoolPairInfo {
   status: boolean
   idTokenA: string
   idTokenB: string
+  nameTokenA: string
+  nameTokenB: string
```

Comment:
```suggestion
```

this file shouldn't be edited since it's internal

Reply:
You're right, I don't even have to change this. Removed now.